src/store/modules/util/actions.ts

```diff
@@ -62,6 +64,45 @@ const actions: ActionTree<UtilState, RootState> = {
     commit(types.UTIL_FACILITY_TYPES_UPDATED, facilityTypes)
   },
 
+  async fetchRoles({ commit, state }) {
+    if (state.roles.length) {
+      return
+    }
+
+    let roles = []
+    const params = {
+      inputFields: {
+        roleTypeGroupId: 'FACILITY_PARTY_ROLE'
+      },
+      viewSize: 100,
+      entityName: 'RoleTypeGroupMemberAndRoleType',
+      orderBy: 'sequenceNum',
+      filterByDate: 'Y',
+      fieldList: ['roleTypeId', 'description']
+    }
+
+    try {
+      const resp = await UtilService.fetchRoles(params)
+      if (!hasError(resp)) {
+        roles = resp.data.docs
+
+        // pushing none explicitly to show on UI
+        roles.push({
+          roleTypeId: 'none',
+          parentTypeId: 'none',
+          description: 'None',
+        })
+      } else {
+        throw resp.data
+      }
+    } catch (error) {
+      showToast(translate('Something went wrong'));
```

Comment:
We should not show toast when fetching additional information or helper information.